packages/vaadin-date-picker/test/wai-aria.test.js

```diff
@@ -2,52 +2,52 @@ import { expect } from '@esm-bundle/chai';
 import sinon from 'sinon';
 import { aTimeout, fixtureSync, isIOS, nextFrame } from '@vaadin/testing-helpers';
 import { IronA11yAnnouncer } from '@polymer/iron-a11y-announcer/iron-a11y-announcer.js';
-import '../vaadin-date-picker.js';
 import { activateScroller, getDefaultI18n, open } from './common.js';
+import '../src/vaadin-date-picker.js';
 
 describe('WAI-ARIA', () => {
   describe('date picker', () => {
-    let datepicker, toggleButton;
+    let datepicker, toggleButton, input;
 
     beforeEach(() => {
-      datepicker = fixtureSync(`<vaadin-date-picker label="ariatest"></vaadin-date-picker>`);
+      datepicker = fixtureSync(`<vaadin-date-picker></vaadin-date-picker>`);
       toggleButton = datepicker.shadowRoot.querySelector('[part="toggle-button"]');
+      input = datepicker.inputElement;
     });
 
-    it('should have application role on input container', () => {
+    // TODO: clarify if this is still needed
+    it.skip('should have application role on input container', () => {
```

Comment:
Doesn't seem to make a difference on JAWS. The current date-picker (before the slotted input change) doesn't seem to benefit from this either.

Reply:
TetraLogical has recommended us to not use `role="application"` so probably we can remove it.

Reply:
Done